frameworks/person-escort-record/questions/terrorism-offences.yml

```diff
@@ -0,0 +1,15 @@
+type: radio
+question: Have they committed any current or previous offences connected to terrorism?
+description: Terrorism offences
+hint: This includes relevant terrorism offences, as well as any offences with a connection to terrorism
+options:
+  -
+    label: "Yes"
+    value: "Yes"
+  -
+    label: "No"
+    value: "No"
+validations:
+  -
+    type: required
+    message: Select yes if they have committed any offences connected to terrorism
```

Comment:
I think the pattern I've used elsewhere uses `the person` instead of `they` in the error message. Might be better to stay consistent for now.

Reply:
amended although there is a question around using `the person` vs `they` as the questions themselves use `they`